### PR TITLE
Mitigate full path presence in exported files (FBX-485)

### PR DESCRIPTION
**Pull request by mfe**
## Purpose of this PR

**Ticket/Jira #:** FBX-485

- Detroy some properties containing absolute paths that are not needed once the structure is created
- Add a warning to warn users Texture workflow is based on absolute paths and this means absolute paths are baked in the exported file when refering to Texture files.
- Add the option to hide the warning in Project settings 

Unrelated to the scope of FBX-485 but dev QoL improv:
- Exposed the Verbose option in Project settings
- Add the PR template in this repo

## Testing

**Functional Testing status:**

- [x] QA pass - all good
- [x] Additional tests with Blender, Maya, FBX Preview and Unity around the texture workflow (ascii / bin / embeded...).

**Performance Testing status:**

## Overall Product Risks

**Complexity:**

Low 
**Halo Effect:**

Medium
According to Autodesk, Url and LastUrl are meaningful only with the FbxXRefManager, so they are fine to destroy if the goal is to hide the absolute path. But you don't know what you don't know.
Functional testing has been done, to mitigate the risk

## Documentation & UX Writing

| User facing text to review |  | Details |
| :--- | :--- | :--- |
| User interface | Y  |  Project settings |
| Public API docs | N  |  |
| Changelog | Y |  |

| Documentation halo effect |  | Jira link |
| :--- | :--- | :--- |
| User manual | Y |  updates to review |
| Other docs |  |  |

## Additional information

**Note to reviewers:**

**Reminder:**

- [x] Add entry in CHANGELOG.md

### CODE REVIEW

**Review by lassond — APPROVED** [review]

File: com.unity.formats.fbx/Editor/FbxExporter.cs [hunk 1]
```diff
@@ -695,6 +695,12 @@ internal bool ExportTexture(Material unityMaterial, string unityPropName,
                 fbxTexture.SetWrapMode(GetWrapModeFromUnityWrapMode(wrapModeU, unityMaterial.name, unityPropName),
                     GetWrapModeFromUnityWrapMode(wrapModeV, unityMaterial.name, unityPropName));
                 TextureMap.Add(tuple, fbxTexture);
+                if (!ExportSettings.DisableAbsolutePathWarningProperty)
+                    Debug.LogWarning(
+                        "The source object contains a reference to a texture file. As a consequence, the exported FBX will contain " +
+                        "an absolute reference to this texture file.\nIf you plan to share this FBX asset, double check your path does " +
+                        "not contain sensitive information.\nThis warning can be disabled in the Fbx Export section of your Project Settings." +
```
Comment (sebastienduverne) [inline com.unity.formats.fbx/Editor/FbxExporter.cs:702]:
```suggestion
                        "The source object contains a reference to a texture file. As a consequence, the exported FBX will contain " +
                        "an absolute reference to this texture file.\nIf you plan to share this FBX asset, make sure your path does " +
                        "not contain sensitive information.\nYou can disable this warning in the Fbx Export section of your Project Settings." +
```

File: com.unity.formats.fbx/Documentation~/ref-project-settings.md
Comment (sebastienduverne) [inline com.unity.formats.fbx/Documentation~/ref-project-settings.md:1]: The screenshot referenced in this page is outdated, I'm going to push an updated one in this PR.

**Review by sebastienduverne — COMMENTED** [review]
A few suggestions in changelog, docs and UI message.
I'm also going to provide an updated Project Settings screenshot.

**Review by sebastienduverne — APPROVED** [review]
LGTM

**Pull request merged**